tests/regression_tests/volume_calc/test.py

```diff
@@ -8,6 +8,19 @@
 
 
 class VolumeTest(PyAPITestHarness):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.exp_std_dev = 1e-01
+        self.std_dev_iters = 521
+
+        self.exp_rel_err = 1e-01
+        self.rel_err_iters = 10
+
+        self.exp_variance = 5e-02
+        self.variance_iters = 105
```

Comment:
Not a fan of these hard coded results as part of the Python code. This means that if, e.g., a random number stream changes in OpenMC, we'd have to manually change these. Can you just write these out as part of the tests `_get_results` / `_write_results`?

Reply:
I guess the `exp_*` variables are alright, but the other ones would cause problems

Reply:
Good point. Those are much better off in the results file. I kept the `exp_*` variables in `test.py` mostly to explicitly test the PythonAPI and its reader/writer.